qprofiler2/src/org/qcmg/qprofiler2/util/SummaryReportUtils.java

```diff
@@ -82,10 +82,11 @@ public static void tallyQualScores(byte [] data, QCMGAtomicLongArray array) {
 			 * @return: the minimum, maximum, mean, mode, medium, sum of array elements ,  sum of array elements multiply it's index position;
 			 */
 			public   TallyStats(QCMGAtomicLongArray array) {
+								
 				long arrayLength = null != array ? array.length() : 0;
 				
 				long bases = 0,counts = 0;		
-				for (int i = 1 ; i < arrayLength ; i++){
+				for (int i = 0; i < arrayLength ; i++){
```

Comment:
are there tests covering this?

Reply:
unMappedReadTest()
 noSeqReadTest()
cover above case